typing/env.ml

```diff
@@ -712,7 +712,11 @@ let check_consistency ps =
               add_import name;
               Consistbl.check crc_units name crc ps.ps_filename)
       ps.ps_crcs;
-  with Consistbl.Inconsistency(name, source, auth) ->
+  with Consistbl.Inconsistency {
+      unit_name = name;
+      inconsistent_source = source;
+      original_source = auth;
+    } ->
     error (Inconsistent_import(name, auth, source))
```

Comment:
Note that here, the order of arguments is exchanged between `Inconsistency` and `Inconsistent_import`. This was already here in the original code. I don't know if the order of `Inconsistency_import` is intentional or a small mistake, but the semantics of the two arguments is the same (the error messages just says "foo is distinct from blah").